src/Promitor.Agents.Scraper/Scheduling/ResourcesScrapingJob.cs

```diff
@@ -251,22 +251,63 @@ private void GetResourceScrapeDefinition(IAzureResourceDefinition resourceDefini
         }
 
         private async Task ScrapeMetrics(IEnumerable<ScrapeDefinition<IAzureResourceDefinition>> scrapeDefinitions, CancellationToken cancellationToken)
-        {
+        {   
             var tasks = new List<Task>();
+            var batchScrapingEnabled = this._azureMonitorIntegrationConfiguration.Value.MetricsBatching?.Enabled ?? false;
+            Logger.LogInformation("Parsed batch config: {Enabled}, {BatchSize}",  this._azureMonitorIntegrationConfiguration.Value.MetricsBatching.Enabled, this._azureMonitorIntegrationConfiguration.Value.MetricsBatching.MaxBatchSize);
+            Logger.LogInformation("Parsed SDK runtime config {Enabled}",  this._azureMonitorIntegrationConfiguration.Value.UseAzureMonitorSdk);
```

Comment:
Can we consolidate this with the log above?

Reply:
Don't think we need this actually, it was just a logging statement I used for debugging